### PR TITLE
Feature/rst 973 emergency application

### PR DESCRIPTION
**Pull request by zaparka**
This is PR from me and @temi77. This is the first story of many for EV check. That's the reason why this is not merging into master but into a separate branch.

### CODE REVIEW

File: spec/support/admin_application_steps.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ def fill_personal_details(ni_number = 'SN123456C')
 end
 
 def fill_application_details
-  expect(page).to have_text 'Application details'
+  expect(page).to have_css("h2", text: "Application details")
```
Comment (AlexaVB) [inline spec/support/admin_application_steps.rb]: Might be good for consistency to keep to single quotes?

File: spec/services/evidence_check_selector_spec.rb [hunk 2]
```diff
@@ -39,6 +39,57 @@
...
+    describe 'should skip EV check' do
+      let(:application) { instance_spy 'Application', outcome: 'full', application_type: 'income' }
+      let(:detail) { build_stubbed :detail }
+
+      before do
+        allow(evidence_check_selector).to receive(:evidence_check_type).and_return 'test'
+        allow(application).to receive(:detail).and_return detail
+      end
+
+      it "creates EV check" do
```
Comment (AlexaVB) [inline spec/services/evidence_check_selector_spec.rb]: Might be good for consistency to keep to single quotes?

File: spec/features/no_evidence_check/rst_973_spec.rb [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    end
+
+    scenario 'Every 10th application is not evidence check for emergency application' do
+      start_new_application
+      fill_personal_details
+      fill_application_emergency_details
+      fill_saving_and_investment
+      fill_benefits(false)
+      fill_income(false)
+      click_button "Complete processing"
```
Comment (AlexaVB) [inline spec/features/no_evidence_check/rst_973_spec.rb]: Might be good for consistency to keep to single quotes?

File: spec/features/no_evidence_check/rst_973_spec.rb [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+    before do
+      create_list :application_full_remission, 1, :refund
+    end
+
+    scenario 'Every 2nd application is not evidence check for emergency application' do
+      start_new_application
+      fill_personal_details
+      check 'This is an emergency case'
+      fill_in 'Reason for emergency', with: "I'm in hurry"
```
Comment (AlexaVB) [inline spec/features/no_evidence_check/rst_973_spec.rb]: Might be good for consistency to keep to single quotes?

File: spec/features/no_evidence_check/rst_973_spec.rb [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+      fill_application_refund_details
+      fill_saving_and_investment
+      fill_benefits(false)
+      fill_income(false)
+      click_button "Complete processing"
```
Comment (AlexaVB) [inline spec/features/no_evidence_check/rst_973_spec.rb]: Might be good for consistency to keep to single quotes?

File: spec/features/no_evidence_check/rst_973_spec.rb [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+        fill_benefits(false)
+        fill_income(false)
+
+        expect(page).to have_text 'Check details'
+        click_button 'Complete processing'
+
+        visit home_index_url
+        create_flag_check('SN123456D')
+
+        click_button "Start now"
```
Comment (AlexaVB) [inline spec/features/no_evidence_check/rst_973_spec.rb]: Might be good for consistency to keep to single quotes?

Comment (temi77) [reply spec/features/no_evidence_check/rst_973_spec.rb]: All Noted will apply fix

File: spec/features/no_evidence_check/rst_973_spec.rb (hunk 5 above)
Comment (temi77) [reply spec/features/no_evidence_check/rst_973_spec.rb]: Thanks, will apply fix across board

**Pull request merged**